game/SwitchDB_Miidatafile.ksy

```diff
@@ -0,0 +1,157 @@
+meta:
+  id: miidata_sdb
+  endian: le

```

Comment:
Like with the previous spec, there's no need to set an endianness, because none of the integers are multiple bytes large.

Though because the spec uses bit-sized integers, you should set the bit order using `bit-endian: be`. If you don't set `bit-endian`, the compiler also defaults to `be`, but to avoid confusion it's recommended to always set `bit-endian` explicitly if possible. (In the [`bit-endian` docs](https://doc.kaitai.io/user_guide.html#bit-endian) there's a detailed explanation why, if you're interested.)
```suggestion
  bit-endian: be
```